br.gov.cti.invesalius.json

```diff
@@ -0,0 +1,203 @@
+{
+    "app-id": "br.gov.cti.invesalius",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.gfortran-62"],
+    "command": "runner.sh",
+    "cleanup": [
+        "/include",
+        "/lib/cmake",
+        "/share/cmake",
+        "/share/doc",
+        "/share/swig",
+        "/bin/cy*",
+        "/bin/nib*",
+        "/bin/py*",
+        "/bin/skivi",
+        "/bin/swig",
+        "/bin/vtk*",
+        "/bin/wx*",
+        "/bin/*.py",
+        "/bin/f2py3",
+        "/bin/helpviewer",
+        "/bin/img*",
+        "/bin/parrec2nii"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/gfortran-62/bin"
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=home",
+        "--filesystem=/tmp"
```

Comment:
This won't work, see http://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access

Reply:
I don' t know if I understood it correctly. I need home, to user open dicom files from home folder and to save the projects there. I also added "--device=all" because user may want to open dicom files from cd-rom, dvd-rom or an usb-driver. I deleted "--socket=pulseaudio" because InVesalius doesn't need to play any sound.